bin/update_pythons.py

```diff
@@ -227,6 +228,7 @@ def update_config(self, config: Dict[str, str]) -> None:
         orig_config = copy.copy(config)
         config_update: Optional[AnyConfig]
 
+        # We need to use ** in update due to MyPy (probably a bug)
```

Comment:
MyPy's handling of TypedDict's is pretty buggy. I had to do this, and I also don't know how to type narrow on a Union of TypedDicts, as `assert isinstance` is not allowed, so this takes a normal Dict for now. 🤷